schemacrawler: Add version 16.16.5

sualeh opened the pull request.

sualeh commented:
@tech189  - any ideas on why the validation is failing? The manifest works locally for me.

sualeh commented:
@tech189  - are you fine with this change?

tech189 commented on bucket/schemacrawler.json:
```diff
@@ -0,0 +1,17 @@
+{
+  "version": "16.16.5",
+  "description": "SchemaCrawler - Free database schema discovery and comprehension tool",
```
```suggestion
  "description": "Database schema discovery and comprehension tool",
```
No need to repeat name in description.

sualeh replied:
I have removed the product name from the description but left the tag line is intact.

tech189 commented:
/verify

tech189 commented:
Hmm could you add the hash to autoupdate please? You publish it here: https://github.com/schemacrawler/SchemaCrawler-Installers/releases/download/v16.16.5/SchemaCrawler-16.16.5.msi.SHA256

sualeh commented:
/verify

sualeh commented:
@tech189 - this one is a go too - what do you think?

tech189 commented on bucket/schemacrawler.json:
```diff
@@ -0,0 +1,21 @@
+    "github": "https://github.com/schemacrawler/SchemaCrawler"
+  },
+  "autoupdate": {
+    "url": "https://github.com/schemacrawler/SchemaCrawler-Installers/releases/download/v$version/SchemaCrawler-$version",
```
```suggestion
    "url": "https://github.com/schemacrawler/SchemaCrawler-Installers/releases/download/v$version/SchemaCrawler-$version.msi",
```
You forgot the extension, it's a 404 when manifest autoupdates

tech189 commented on bucket/schemacrawler.json:
```diff
@@ -0,0 +1,21 @@
+  "autoupdate": {
+    "url": "https://github.com/schemacrawler/SchemaCrawler-Installers/releases/download/v$version/SchemaCrawler-$version",
+    "hash": {
+      "url": "https://github.com/schemacrawler/SchemaCrawler-Installers/releases/download/v$version/SchemaCrawler-$version.msi.SHA256",
```
```suggestion
      "url": "$url.SHA256",
```
Can be simplified!

rashil2000 commented:
- 64bit or 32bit?
- Why is extract_to required?

sualeh commented:
/verify

sualeh commented:
/verify

sualeh commented:
> * 64bit or 32bit?
> * Why is extract_to required?

@rashil2000 
- I have added an architecture section, to be explicit about the bit version
- I have removed extract_to 
Please take a look. Thanks.

rashil2000 commented on bucket/schemacrawler.json:
```diff
@@ -0,0 +1,27 @@
+      "hash": "89726fd5adbcb31cb0476873318e90f09b081c2ecd998a19ec6b0d197266aac4"
+    }
+  },
+  "bin": "SchemaCrawler\\SchemaCrawler.exe",
```
```suggestion
  "extract_dir": "SchemaCrawler",
  "bin": "SchemaCrawler.exe",
```

sualeh replied:
That will not work, @rashil2000 - an extra SchemaCrawler directory is created by extracting the MSI.

sualeh replied:
Sorry - you are right. I am going to make that change. Do I need to make it in the updates section as well?

rashil2000 replied:
Yes, this is for that directory itself. Try it.

rashil2000 reviewed (changes requested):
Reformat the file to use 4 spaces as tab instead of 2

sualeh commented:
@rashil2000 - comments addressed in the latest checkin:

1. 4-space tabs
2. Use an extract dir

rashil2000 commented:
/verify
